internal/winapi/orapi.go

```diff
@@ -0,0 +1,36 @@
+package winapi
```

Comment:
I'd prefer if this was just registry.go, offlinereg.go or a variation of this. Maybe `or` doesn't having a mapping in my head as offline registry yet though haha

Reply:
Renamed to offlinereg.go.